client/src/components/contacts/form-general.vue

```diff
@@ -74,23 +91,26 @@ export default {
       group_id: null,
       name: null,
       source_id: null,
-      first_name: null,
-      last_name: null,
-      type_id: null
+      company_name: null,
+      position: null,
+      type_id: null,
+      status_id: 'NEW'
```

Comment:
This has to be in API, not the JS code

Reply:
We can create contact this every status, not only NEW.